### ISSUE TITLE
wallet crashed when receiving part

### ISSUE THREAD

**Issue by pciavald** [particl/particl-desktop#727]
we were doing a test with @vikas-cis and he sent me 2 tPART, when I received it I seen the notification and the wallet crashed right away with this output on the terminal console : 

```
2018-01-19 16:19:36 connect() to 196.25.195.98:51938 failed after select(): Connection refused (111)
2018-01-19 16:19:39 AddToWallet adc9feb6aec404b123ef5beb0aa6b9ceac58fd08dda676c4f9c01a8948711788  new
[14229:0119/171940.754195:FATAL:wrappable.cc(27)] Check failed: !wrapper_.IsEmpty(). 
#0 0x000002bf6847 <unknown>
#1 0x000002bf4cfb <unknown>
#2 0x000003e91eca mate::WrappableBase::GetWrapper()
#3 0x000003ec5331 _ZN4mate12EventEmitterIN4atom3api12NotificationEE14EmitWithSenderIJEEEbRKN4base16BasicStringPieceISsEEPN7content11WebContentsEPN3IPC7MessageEDpRKT_
#4 0x000003ec2fc8 atom::api::Notification::NotificationDestroyed()
#5 0x000003fcc3e0 brightray::Notification::~Notification()
#6 0x000003fcc9a9 brightray::LibnotifyNotification::~LibnotifyNotification()
#7 0x000003fbcc10 brightray::NotificationPresenter::RemoveNotification()
#8 0x7f5006aed6f5 g_closure_invoke
#9 0x7f5006b010b0 <unknown>
#10 0x7f5006b05696 g_signal_emit_valist
#11 0x7f5006b06920 g_signal_emit
#12 0x7f4fd72c88bd <unknown>
#13 0x7f500000b1c8 ffi_call_unix64
#14 0x7f500000ac2a ffi_call
#15 0x7f5006af684b g_cclosure_marshal_generic
#16 0x7f5006aed6f5 g_closure_invoke
#17 0x7f5006b010b0 <unknown>
#18 0x7f5006b05696 g_signal_emit_valist
#19 0x7f5006b06920 g_signal_emit
#20 0x7f5006fdd31f <unknown>
#21 0x7f5007052ea9 <unknown>
#22 0x7f5003ec00be g_main_context_dispatch
#23 0x7f5003ec1f69 <unknown>
#24 0x7f5003ec1fae g_main_context_iteration
#25 0x000002be7e96 <unknown>
#26 0x000002c602d7 <unknown>
#27 0x000002c6db6e <unknown>
#28 0x000000eb2bcc <unknown>
#29 0x000000eb2a15 <unknown>
#30 0x000000ed2b3d <unknown>
#31 0x0000012f853e <unknown>
#32 0x000000c7905c <unknown>
#33 0x000000c794b0 <unknown>
#34 0x000003e3da88 main
#35 0x7f50017c3f4a __libc_start_main
#36 0x0000005b0e99 <unknown>

[1]    14229 abort (core dumped)  ./packages/linux-unpacked/particl
```

After restarting the wallet, daemon was still running from the previous session, so the crash is from the GUI, not the daemon.

**Comment by vikas-cis** [comment]
I never seen wallet crash on notification.

**Comment by pciavald** [comment]
maybe a track to fix the issue by @kewde : https://github.com/electron/electron/issues/7490

**Comment by vikas-cis** [comment]
@pciavald I think the best https://github.com/electron/electron/issues/10527 is here 👍

**Comment by kewde** [comment]
Need to update electron to 1.8.x for this to go away

**Comment by HenkSwardt** [comment]
I see we have a task logged to upgrade to 1.8 (#748) , but it is earmarked for 1.2 release. Does this mean we will not be able to get to this issue before then?

**Comment by kewde** [comment]
@HenkSwardt 

1.2

**Comment by pciavald** [comment]
should be fixed by #813
